packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -13,6 +13,11 @@
   white-space: nowrap;
   cursor: default;
   user-select: none;
+  display: flex;
+  flex-wrap: nowrap;
+  max-width: 100%;
+  position: absolute;
+  right: 2px;
```

Comment:
I don't think this is quite right. you need at least top/left/right defined if you are going to position absolute it. Also, that doesn't make sense as it's not relative to anything (you can't default the root element to position absolute...